jupyterlab/staging/templates/partial.html

```diff
@@ -14,6 +14,6 @@
   </script>
 
   {% block favicon %}
-  <link rel="icon" type="image/x-icon" href="{{ base_url | escape }}static/favicons/favicon.ico" class="idle favicon">
-  <link rel="" type="image/x-icon" href="{{ base_url | escape }}static/favicons/favicon-busy-1.ico" class="busy favicon">
+  <link rel="icon" type="image/x-icon" alt="idle icon" href="{{ base_url | escape }}static/favicons/favicon.ico" class="idle favicon">
+  <link rel="" type="image/x-icon" alt="busy icon" href="{{ base_url | escape }}static/favicons/favicon-busy-1.ico" class="busy favicon">
```

Comment:
Same reason for captialization

```suggestion
  <link rel="icon" type="image/x-icon" alt="Idle icon" href="{{ base_url | escape }}static/favicons/favicon.ico" class="idle favicon">
  <link rel="" type="image/x-icon" alt="Busy icon" href="{{ base_url | escape }}static/favicons/favicon-busy-1.ico" class="busy favicon">
```